src/solutions/device-management.njk

```diff
@@ -0,0 +1,95 @@
+---
+layout: solution
+title: Device Management for Node-RED Deployments
+subtitle:
+heroImg: ./images/pictograms/edge.png
+description:
+    <p>Companies often deploy Node-RED to edge devices (equipment, gateways, PLCs, etc.) to monitor and collect data from these devices. How to manage deployments 
+    to a large number of devices can be a challenge.  Luckily, FlowForge makes it possible to manage Node-RED deployments out to fleets of devices
+    in a secure and reliable manner.</p>
+---
+
+<div>
+    <div class="-mt-28 text-center">
+        <a href="/contact-us" class="inline-block ff-btn ff-btn--secondary shadow">GET IN TOUCH</a>
```

Comment:
In a follow up PR we should add a 'Book a Demo' Hubspot popup to book one. WDYT?

Reply:
How about we just change to Get in Touch to be Book a Demo?